### PR TITLE
Add Airlock network diagram to docs

### PR DESCRIPTION
**Pull request by yuvalyaron**
## What is being addressed

* Added a diagram that describes Airlock's networking structure
* Fixed bullets in Airlock documentation

### CODE REVIEW

File: docs/azure-tre-overview/airlock.md [hunk 1]
```diff
@@ -141,13 +141,15 @@ graph LR
...
+
 * `stalimex` - storage (st) airlock (al) import (im) external (ex)
 * `stalimip` - storage (st) airlock (al) import (im) in-progress (ip)
 * `stalimrej` - storage (st) airlock (al) import (im) rejected (rej)
 * `stalimblocked` - storage (st) airlock (al) import (im) blocked
 * `stalexapp` - storage (st) airlock (al) export (ex) approved (app)
 
-Workspace
+Workspace:
+
```
Comment (yuvalyaron) [inline docs/azure-tre-overview/airlock.md:152]: adding the line breaks to fix the bullets, as today all of them are showing as 1 line

**Review by eladiw — APPROVED** [review]
lgtm. talked f2f

**Review by joalmeid — APPROVED** [review]
Great work @yuvalyaron .
You found a clear way to place the storage accounts, highlight the networking subnets used and even clearly place what are those accessible from public internet.
It brings clarity!

**Pull request merged**